pkg/ui/ccl/src/views/clusterviz/containers/map/mapLayout.styl

```diff
@@ -0,0 +1,13 @@
+/* Copyright 2017 The Cockroach Authors.
```

Comment:
2018

Reply:
Eh, its a cherry pick, it's fine.

Reply:
ha! probably all the new clusterviz code says 2017. whoops!